packages/strapi/lib/configuration/hooks/core/router/index.js

```diff
@@ -242,25 +234,38 @@ module.exports = strapi => {
         // Add the `responsesPolicy`.
         policies.push(responsesPolicy);
 
-        // Allow string instead of array of policies
+        // Allow string instead of array of policies.
         if (!_.isArray(_.get(value, 'config.policies')) && !_.isEmpty(_.get(value, 'config.policies'))) {
           value.config.policies = [value.config.policies];
         }
 
         if (_.isArray(_.get(value, 'config.policies')) && !_.isEmpty(_.get(value, 'config.policies'))) {
           _.forEach(value.config.policies, policy => {
-            // Looking for global policy or namespaced
-            if (_.startsWith(policy, '*', 0) && !_.isEmpty(_.get(strapi.policies, policy.substring(1).toLowerCase()))) {
-              return policies.push(strapi.policies[policy.substring(1).toLowerCase()]);
-            } else if (!_.startsWith(policy, '*', 0) && !_.isEmpty(_.get(strapi.api, currentApiName + '.policies.' + policy.toLowerCase()))) {
-              return policies.push(strapi.api[currentApiName].policies[policy.toLowerCase()]);
+            // Define global policy prefix.
+            const globalPolicyPrefix = 'global.';
+            const pluginPolicyPrefix = 'plugins.';
+            const policySplited = policy.split('.');
+
+            // Looking for global policy or namespaced.
+            if (_.startsWith(policy, globalPolicyPrefix, 0) && !_.isEmpty(strapi.config.policies, policy.replace(globalPolicyPrefix, ''))) {
+              // Global policy.
+              return policies.push(strapi.config.policies[policy.replace(globalPolicyPrefix, '').toLowerCase()]);
+            } else if (_.startsWith(policy, pluginPolicyPrefix, 0) && strapi.plugins[policySplited[1]] && !_.isEmpty(_.get(strapi.plugins, policySplited[1] + '.config.policies.' + policySplited[2].toLowerCase()))) {
+              // Plugin's policies can be used from app APIs with a specific syntax (`plugins.pluginName.policyName`).
+              return policies.push(_.get(strapi.plugins, policySplited[1] + '.config.policies.' + policySplited[2].toLowerCase()));
+            } else if (!_.startsWith(policy, globalPolicyPrefix, 0) && plugin && !_.isEmpty(_.get(strapi.plugins, plugin + '.config.policies.' + policy.toLowerCase()))) {
+              // Plugin policy used in the plugin itself.
+              return policies.push(strapi.plugins[plugin].config.policies[policy.toLowerCase()]);
```

Comment:
To keep consistency in our codebase, it could be great to use the Lodash `get` method or remove it but always use the same syntax for returning the policy function.